src/util/webSocket.edgeworker.ts

```diff
@@ -1,9 +1,9 @@
 export function wsEdgeWorker(url: string) {
-  const webSocketUrl = url.replace('wss:', 'https:');
+  const webSocketUrl = url.replace("wss:", "https:");
   // @ts-ignore
-  const [client, server] = Object.values(new WebSocketPair());
+  const [client, server]: any = Object.values(new WebSocketPair());
```

Comment:
can we get rid of any ?

Reply:
The new version of Typescript wants type for `client` and `server`. I have tried to add Cloudflare types to the project but it clashes with other packages. I need to find some other solution how to add Cloudflare types but until that this is ignored as well because `WebSocketPair()` does not have type as well 🥲

Reply:
@dlozina-macrometa can you check this out to see if this provides any solution for the above problem:
https://github.com/cloudflare/workers-types/issues/84

Reply:
Yes @koshy-thomas I already tried to do that. They have types, but I found out that they do not play well with other parts of our SDK 😢 I will revisit this when we have time.